apps/extension/src/background/ledger/ledger.ts

```diff
@@ -90,6 +90,30 @@ export class Ledger {
     };
   }
 
+  public async showAddressAndPublicKey(
```

Comment:
Nitpick: once we’re not triggering showing / displaying anything, wouldn’t be better to have this method named as retrieveAddressAndPublicKey? Or getAddressOrPublicKey?

Reply:
we already have `getAddressAndPublicKey`, which simply retrieves the key. I'm leaving that one in there as it might be used in the signing process still (trying not to break anything)

I can rename it to `retrieveAddressAndPublicKey`!

Reply:
I'm gonna make the call to update this in the future. I'll add this to https://github.com/anoma/namada-interface/pull/454, that way I can remove the other call as well